xbmc/cores/AudioEngine/Sinks/AESinkPULSE.cpp

```diff
@@ -280,6 +280,17 @@ struct SinkInfoStruct
   }
 };
 
+struct ModuleInfoStruct
+{
+  pa_threaded_mainloop *mainloop;
+  bool hasAllowPT;
+  ModuleInfoStruct()
+  {
+    mainloop = NULL;
```

Comment:
`nullptr`

Reply:
In general pulseaudio is a C API. Therefore I decided to keep the coding standards this file already has.